var/spack/repos/builtin/packages/flit/package.py

```diff
@@ -3,27 +3,32 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-
 from spack import *
 
-
 class Flit(MakefilePackage):
     """Floating-point Litmus Tests (FLiT) is a C++ test infrastructure for
     detecting variability in floating-point code caused by variations in
     compiler code generation, hardware and execution environments."""
 
     homepage = "https://pruners.github.io/flit"
     url      = "https://github.com/PRUNERS/FLiT"
-    url      = "https://github.com/PRUNERS/FLiT/archive/v2.0-alpha.1.tar.gz"
+    url      = "https://github.com/PRUNERS/FLiT/archive/v2.1.0.tar.gz"
 
+    version('2.1.0', sha256='b31ffa02fda1ab0f5555acdc6edc353d93d53ae8ef85e099f83bcf1c83e70885')
     version('2.0-alpha.1', sha256='8de2bd400acf0f513d69f3dbf588e8984dfb18b8ccaaf684391811a0582f694b')
```

Comment:
You can probably remove this alpha release now that a stable release version is out.

Reply:
Done